src/devices/bus/spectrum/mface.cpp

```diff
@@ -24,37 +97,77 @@ DEFINE_DEVICE_TYPE(SPECTRUM_MPRINT, spectrum_mprint_device, "spectrum_mprint", "
 //  INPUT_PORTS( mface )
 //-------------------------------------------------
 
-INPUT_PORTS_START(mface)
+INPUT_PORTS_START( mface )
 	PORT_START("BUTTON")
-	PORT_BIT(0x01, IP_ACTIVE_LOW, IPT_BUTTON1) PORT_NAME("Multiface") PORT_CODE(KEYCODE_F12) PORT_CHANGED_MEMBER(DEVICE_SELF, spectrum_mface1_device, magic_button, 0)
+	PORT_BIT(0x01, IP_ACTIVE_LOW, IPT_BUTTON1) PORT_NAME("Red Button") PORT_CODE(KEYCODE_BACKSPACE) PORT_CHANGED_MEMBER(DEVICE_SELF, spectrum_mface_base_device, magic_button, 0)
```

Comment:
Why the change to backspace here?

Reply:
it F12 conflicts with debugger use, which is quite a major problem, I spent ages trying to chase down another issue until I realised F12, to exit debugger, was breaking things.

Reply:
Yes it was both that and simply that F12 is "snapshot" button by default in Mame, backspace seems to have no default mapping.